sample/build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins {
+  id("com.android.application")
+  kotlin("android")
+}
+
+android {
+  compileSdkVersion(28)
+  buildToolsVersion = "29.0.2"
+
+  compileOptions {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+  }
+
+  defaultConfig {
+    minSdkVersion(17)
+    targetSdkVersion(28)
+    applicationId = "com.squareup.radiography.sample"
```

Comment:
I'm guessing this is intentional since this exists in a more universal namespace, but just wanted to confirm this shouldn't just be `radiography.sample`.

Reply:
Yep felt like apps should use the more universal namespace, though honestly 🤷‍♂️